specs/phase1/beacon-chain.md

```diff
@@ -476,7 +476,7 @@ def get_online_validator_indices(state: BeaconState) -> Set[ValidatorIndex]:
 ```python
 def get_shard_committee(beacon_state: BeaconState, epoch: Epoch, shard: Shard) -> Sequence[ValidatorIndex]:
     source_epoch = epoch - epoch % SHARD_COMMITTEE_PERIOD
-    if source_epoch > 0:
+    if source_epoch > SHARD_COMMITTEE_PERIOD:
         source_epoch -= SHARD_COMMITTEE_PERIOD
```

Comment:
@djrtwo @terencechain do you remember why we have to adjust `source_epoch` to the start of the previous period?

Reply:
I wanted to ask that as well 😅

Reply:
to provide `SHARD_COMMITTEE_PERIOD` lookahead on the assignment.

The idea is to know which shard you headed to and have `period` length to get there.